tempesta_fw/http_msg.c

```diff
@@ -45,16 +45,23 @@ tfw_http_msg_hdr_val(TfwStr *hdr, int id, TfwStr *val)
 	TfwStr *c;
 	int nlen = hdr_lens[id];
 
-	BUG_ON(TFW_STR_PLAIN(hdr));
```

Comment:
I'd create a specific `BUG_ON()` rather than removing it completely. For instance, in this case the only exception is the `Host:` header. Why not modify the `BUG_ON()` to crash on anything else if a string is plain?